src/components/navBar/navBar.module.scss

```diff
@@ -1,35 +1,217 @@
-$offset: 10px;
-$thickness: 3px;
+.wrapper {
+  font: 700 16px "Roboto", sans-serif;
```

Comment:
Move to variable files all the styling values eg: `font-family`, `base-font-size`, `font-weight`, etc

Reply:
Done